doc/user/user/samples/index.md

```diff
@@ -126,100 +126,75 @@ Adding a new sample
+the [project settings page](../../user/project/#associated-projects)) can have their sample displayed in the table as
+well by selecting the project name within the filter
+dropdown. These samples will be included in all filtering.
 
-To search sample by a date range, click on the date range field.  A drop down will be displayed with pre-determine ranges:
+### Filter by File
 
-* Last 30 Days
-* Last 60 Days
-* Last 120 Days
+Filtering by file allows you to quickly filter the table by use a text (<pr>.txt</pre>) file containing the sample names
+you are looking for. Each sample name must be on its own line.
 
-Or you can enter a custom date range by selecting the dates in the calendar.
-
-![Samples advanced filter dialogue apply.](images/advanced-filter-dialogue-apply.png)
-
-To apply the selected filters click the 'Filter' button.
-
-![Samples advanced filter applied state.](images/advanced-filter-applied-state.png)
-
-Once the filter is applied, the samples table will be updated with the filtered samples.  When an advanced filter is applied, a tag is created below the filter button to allow the user to know what filters are currently applied.  To remove a specific filter click on the tag itself.
-
-### Clearing Filters
-
-![Samples clear filters button.](images/advanced-filter-clear.png)
-
-To clear all currently applied filters and search, click on the clear button to the right of the filter area.
-
-Filtering and Selecting by File
--------------------------------
-
-As projects become larger, it becomes unwieldy to select a large subset of samples.  To facilitate this, there is the 'Filter by File' option.
-
-* Create a `.txt` file that contains the name of each sample you want to select on a new line.  You can either:
-    - Use a text editor like Windows Notepad or TextEdit on Mac (note: when creating a new text document in TextEdit, press cmd + shift + t to change to the `.txt` format)
-    - Or create a spreadsheet in Excel with a single column and save the file as a 'Text (Tab Delimited) (*.txt)' file.
-
-Example (`project_5_filter.txt`):
+E.g.
 
 <pre>
-03-3333
-10-6966
-15-7569
+10-8231
+02-2222
+01-1111
+11-11-1
+11-0002
 </pre>
 
-* On the project samples page, click the <span class="fa fa-file-o fa-fw"></span> button from the filter menu.
-
-![Filter by File Button](images/filter_by_file_btn.png)
+![Filter by file preview.](images/filter-by-file.gif)
 
-* Select the file you created.  The samples will be filtered by these names.
+![Filter by File button](images/filter-by-file-button.png)
 
-If all sample names are found, a green success notification will appear in the upper right corner of the window.  This notification will disappear after 2 seconds.
+To filter the table by file click the "Filter by File" button at the top of the table. This will open a modal window
+which will allow you to select the file containing the list of names to filter by. Once selected, IRIDA will if the
```

Comment:
Should this read `IRIDA will display`

Reply:
Updated in d9a816b58e492403ff3fcf76fd26caff5b893e2f